src/licensedcode/data/rules/any-osi_1.RULE

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I do not think we want this. This is going to trigger a lot of noisy false positive matches.

Reply:
Yes, you're correct, removing this.